peepingtom/io_/write/star/star.py

```diff
@@ -15,31 +15,31 @@ def matrix2euler(rotation_matrices, convention):
         rotation_matrices = rotation_matrices.transpose((0, 2, 1))
 
     euler_angles = eulerangles.matrix2euler(rotation_matrices,
-                                            axes=convention['axes'],
-                                            intrinsic=convention['intrinsic'],
-                                            right_handed_rotation=convention['right_handed_rotation'])
+                                            axes='zyz',
```

Comment:
little bit worries about the lack of separation between star files and the conventions for angles in them but okay to leave it for now as long as everything is working!

Reply:
I'm changing this file quite a bit, so we can come back to this when it's fixed. But are there cases in which relion uses different conventions?